contracts/EverlongStrategy.sol

```diff
@@ -502,27 +502,49 @@ contract EverlongStrategy is BaseStrategy {
                 ""
             );
 
+            // Calculate the amount of bonds needed to reach the minimum
+            // transaction amount.
+            uint256 minimumBonds = minimumTransactionAmount().mulDivUp(
+                position.bondAmount,
+                totalPositionValue
+            );
```

Comment:
when you're closing longs, the minimum tx amount refers to **bond** amounts, you don't need to do any base <> share conversions then. just use `_poolConfig.minimumTransactionAmount`.

See https://github.com/delvtech/hyperdrive/blob/02d3cd465afbf86cc5f490c618a3a850c6dc6bfe/contracts/src/internal/HyperdriveLong.sol#L167-L171:

```solidity
if (_bondAmount < _minimumTransactionAmount) {
    revert IHyperdrive.MinimumTransactionAmount();
}
```

Reply:
So I don't think we even need to do this calculation to convert it to base (or share) units.

Reply:
Got it! Thanks for pointing this out. Included the fix